webapp/data_structures/RELATIONALOP.js

```diff
@@ -1,11 +1,10 @@
 function RELATIONALOP() {
 
     RELATIONALOP.prototype.define = function RELATIONALOP() {
-        //this.ipar = 2;
-        this.label = "<";
-	this.rule=2;
-	this.zcr=0;
-	this.Datatype=1;
+        this.label = "&lt";
```

Comment:
```suggestion
        var label = "&lt;";
```
this is incorrect syntax. Use **&amp;lt;** or **&lt;**.

This applies in the set and import methods also.

Do not use this here.